test/unit/index.test.ts

```diff
@@ -0,0 +1,161 @@
+  ['ServerApiVersion',                   'reason'],
+  ['BSONType',                           'reason'],
+  ['ReturnDocument',                     'reason'],
+  ['ProfilingLevel',                     'reason'],
+  ['ReadConcernLevel',                   'reason'],
+  ['ReadPreferenceMode',                 'reason'],
+  ['ServerType',                         'reason'],
+  ['TopologyType',                       'reason'],
+  ['ReadConcern',                        'reason'],
+  ['ReadPreference',                     'reason'],
+  ['WriteConcern',                       'reason'],
+  // events
+  ['CommandFailedEvent',                 'reason'],
+  ['CommandStartedEvent',                'reason'],
+  ['CommandSucceededEvent',              'reason'],
+  ['ConnectionCheckedInEvent',           'reason'],
+  ['ConnectionCheckedOutEvent',          'reason'],
+  ['ConnectionCheckOutFailedEvent',      'reason'],
+  ['ConnectionCheckOutStartedEvent',     'reason'],
+  ['ConnectionClosedEvent',              'reason'],
+  ['ConnectionCreatedEvent',             'reason'],
+  ['ConnectionPoolClearedEvent',         'reason'],
+  ['ConnectionPoolClosedEvent',          'reason'],
+  ['ConnectionPoolCreatedEvent',         'reason'],
+  ['ConnectionPoolMonitoringEvent',      'reason'],
+  ['ConnectionReadyEvent',               'reason'],
+  ['ServerClosedEvent',                  'reason'],
+  ['ServerDescriptionChangedEvent',      'reason'],
+  ['ServerHeartbeatFailedEvent',         'reason'],
+  ['ServerHeartbeatStartedEvent',        'reason'],
+  ['ServerHeartbeatSucceededEvent',      'reason'],
+  ['ServerOpeningEvent',                 'reason'],
+  ['TopologyClosedEvent',                'reason'],
+  ['TopologyDescriptionChangedEvent',    'reason'],
+  ['TopologyOpeningEvent',               'reason'],
+  ['SrvPollingEvent',                    'reason'],
+
+  ['GridFSBucketReadStream',             'reason'],
+  ['GridFSBucketWriteStream',            'reason'],
+  ['OrderedBulkOperation',               'reason'],
+  ['UnorderedBulkOperation',             'reason'],
+
+  // TS-NODE Adds these keys but they are undefined... weird
+  ['AnyBulkWriteOperation',              'ts-node'],
+  ['BulkWriteOptions',                   'ts-node'],
+]);
+
+describe('mongodb entrypoint', () => {
+  it('should export all keys in our exports list', () => {
+    expect(sorted(Object.keys(mongodb), byStrings)).to.deep.equal(
+      sorted(expectedExports.keys(), byStrings)
+    );
+  });
+
+  it('should export only keys in our exports list', () => {
+    const currentExports = Object.keys(mongodb);
+    const difference = setDifference(currentExports, expectedExports.keys());
+    expect(
+      difference,
+      `Found extra exports [${Array.from(difference).join(
+        ', '
+      )}], if these are expected, just add them to the list in this file`
+    ).to.have.lengthOf(0);
+  });
+
+  it('meta test: ts-node adds keys to the module that point to undefined', () => {
```

Comment:
assuming we figure out why this is happening and that it is correct, we should still keep with the convention for `it` phrasing, i.e., change this to something like `should export keys added by ts-node as undefined`

Reply:
yep, fixed